.github/workflows/evaluation-test-reusable.yaml

```diff
@@ -0,0 +1,193 @@
+name: "Evaluation tests"
+run-name: "Evaluation tests"
+
+on:
+  workflow_call:
+
+# global env variables.
+env:
+  DOCKER_TIMEOUT: 30
+  K3D_VERSION: "v5.7.2" # Required K3d version.
+  REPOSITORY_FULL_NAME: "${{ github.repository }}" # <owner>/<repository-name>.
+  GITHUB_EVENT_ACTION: "${{ github.event.action }}"
+  IMAGE_NAME: "europe-docker.pkg.dev/kyma-project/dev/kyma-companion:PR-${{ github.event.number }}"
```

Comment:
IMO, the image should be a input parameter to this reusable job, because for release images the repo and tag will be different.

Reply:
agree, done